### PR TITLE
Correctly check for user error on creation task

### PR DESCRIPTION
**Pull request by javitonino**
CR @gfiorav 

cc @matallo

### CODE REVIEW

**Comment by matallo** [comment]
LGTM 😅

File: lib/tasks/setup.rake [hunk 1]
```diff
@@ -68,7 +68,7 @@ DESC
         u.builder_enabled = true
       end
 
-      if !user.errors.empty?
+      unless user.valid?
```
Comment (gfiorav) [inline lib/tasks/setup.rake]: What does .valid? source code do? I thought it checked `errors.empty?`

Comment (javitonino) [inline lib/tasks/setup.rake]: It actually runs the validations. `errors` is empty until you call `.save` or `.valid?`.

Comment (javitonino) [inline lib/tasks/setup.rake]: The problem here was that `errors` was being checked before calling `.save` or `.valid?`, so it is always empty.

Comment (gfiorav) [inline lib/tasks/setup.rake]: Oh! Of course!! I didn't even consider that `.valid?` hadn't been run beforehand.

👌

**Pull request merged**